threeD-components/Social.js

```diff
@@ -56,8 +56,8 @@ const Social = (props = {}) => {
     <div class=threeD-social>
       <div class=peer-list>
         ${peers.length > 0 ? peers.map(rig => {
-          return `<a class=peer id=peer name=${rig.peerConnection.connectionId}>
-            <div class=name>${rig.textMesh.text}</div>
+          return `<a class=peer id=peer name=${rig.peerConnection?.connectionId}>
+            <div class=name>${rig.textMesh?.text}</div>
```

Comment:
Can there ever be no `textMesh`? Similar for `peerConnection`.

Reply:
maybe, we don't know